bin/dotd_menu

```diff
@@ -0,0 +1,349 @@
+  # Double-escape the command string, once for each remote shell we're passing
+  # through.
+  system "ssh -t gateway.code.org ssh -t #{server_name} #{command.inspect.inspect}"
+end
+
+# Prints the DTP message, formatted based on its contents.
+def print_dtp_message
+  message = "Slack#developers says DTP: #{DevelopersTopic.dtp}"
+  if DevelopersTopic.dtp?
+    puts dim message
+  else
+    puts red message
+  end
+end
+
+# Prints the files in the pull request that are in our DB migration directories.
+def print_database_changes(pull_request_number)
+  database_changes = GitHub.database_changes(pull_request_number)
+  return if database_changes.empty?
+  database_changes_output = database_changes.join("\n       ")
+  puts <<-EOS.unindent
+
+     #{red 'WARNING: THIS PR CONTAINS DATABASE CHANGES:'}
+
+       #{red database_changes_output}
+
+  EOS
+end
+
+# Retrieves recent Honeybadger issues, displaying them on the console.
+def get_honeybadger_issues
+  issues = Honeybadger.get_recent_issues
+  issues.sort_by! do |issue|
+    [issue[:project], issue[:environment], (issue[:assignee] ? issue[:assignee] : 'ZZZ')]
+  end
+
+  puts '' if issues.count > 0
+  issues.each do |issue|
+    environment =
+      if issue[:environment] == 'production'
+        (red issue[:environment]).to_s
+      else
+        (bold issue[:environment]).to_s
+      end
+    project = bold issue[:project]
+    message = issue[:assignee] ? (dim issue[:message]).to_s : issue[:message]
+    output = <<-EOS.unindent
+      [#{project}, #{environment}]: #{message}
+          #{dim issue[:url]}
+    EOS
+    # TODO(asher): Eliminate this hackiness, for some reason, some extra newlines are showing up at
+    # the end of the output. This seems correlated with environment and project.
+    puts output.strip
+    puts ''
+  end
+  puts '' if issues.count > 0
+end
+
+# Creates a new branch with the given name from the given commit on the test
+# server (without checking it out) and pushes it to GitHub.
+def create_branch_from_commit(branch_name, commit)
+  # We use the test server as a workspace because we don't want to require that
+  # the DOTD script be run from within the repo.
+  # We never actually checkout the created branch so we don't interfere with
+  # anything else test might be doing at the time.
+  run_on(
+    'test',
+    [
+      'cd test',
+      "git branch --no-track #{branch_name} #{commit}",
+      "git push origin #{branch_name}"
+    ].join(' && ')
+  )
+end
+
+# Deletes the specified branch, both from the test server and from GitHub.
+def delete_branch(branch_name)
+  # We use the test server as a workspace because we don't want to require that
+  # the DOTD script be run from within the repo.
+  run_on(
+    'test',
+    [
+      'cd test',
+      "git branch -D #{branch_name}",
+      "git push origin --delete #{branch_name}"
+    ].join(' && ')
+  )
+end
+
+def main
+  check_for_cdo_keys
+
+  puts_script_intro
+  ask_for_name
```

Comment:
It looks like there's not currently any reason to ask for the DOTD name.  We used this to annotate content scoops before.  Should we skip this step now?